index.html

```diff
@@ -3,6 +3,7 @@
 
 <head>
 
+<meta name="viewport" content="width=device-width, initial-scale=1">
 <meta charset="utf-8"/>
 <link rel="stylesheet" href="style.css"/>
 <title>Mobile Portfolio</title>
```

Comment:
- [ ] I kindly suggest that you surround your logo on line 18 with an `a`  tag as it can be useful to get the user back to the home page.

## EXAMPLE
```
  <div class="logo">
      <p><a  href="./index.html">My Logo</a></p>
    </div>
```